v1/lib/core/DocsLayout.js

```diff
@@ -90,15 +93,24 @@ class DocsLayout extends React.Component {
               version={metadata.version}
               language={metadata.language}
             />
-            {this.props.config.enableUpdateTime &&
-              updateTime && (
-                <div className="docLastUpdateTimestamp">
+            {(updateTime || updateAuthor) && (
+              <div className="docLastUpdate">
+                {updateTime && (
```

Comment:
Let's use a combined format:

- Only `enableUpdateTime`: "Last updated on 2018-10-15 01:23"
- Only `enableUpdateBy`: "Last updated by @fiennyangeln"
- Both `enableUpdateTime` and `enableUpdateBy`: "Last updated on 2018-10-15 01:23 by @fiennyangeln"

WDYT?

Reply:
It's nice 😍 ! I will implement this.